File: Iniciante/2029.py
while True:
  try:
    volume = float(input())
    diametro = float(input())

    pi = 3.14
    raio = diametro/2
    altura = volume/(pi*(raio**2))
    area = pi*raio**2

    print('ALTURA = {:.2f}'.format(altura))
    print('AREA = {:.2f}'.format(area))

  except EOFError:
    break

File: Iniciante/2029.js
const pi = 3.14;

while (lines.length > 0) {
  let volume = Number(lines.shift());
  let diametro = Number(lines.shift());
  
  let raio = diametro/2;
  let altura = volume/(pi*(raio**2));
  let area = pi*raio**2;

  console.log(`ALTURA = ${altura.toFixed(2)}`);
  console.log(`AREA = ${area.toFixed(2)}`);
}
